library/core/src/intrinsics.rs

```diff
@@ -712,8 +712,14 @@ extern "rust-intrinsic" {
 
     /// Aborts the execution of the process.
     ///
-    /// A more user-friendly and stable version of this operation is
-    /// [`std::process::abort`](../../std/process/fn.abort.html).
+    /// [`std::process::abort`](../../std/process/fn.abort.html) is to be preferred if possible,
+    /// as its behaviour is more user-friendly and more stable.
```

Comment:
nit: we generally prefer American spelling in the docs.
```suggestion
    /// as its behavior is more user-friendly and more stable.
```